.github/workflows/node.js.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "main", "backend" ]
+  pull_request:
+    branches: [ "main", "backend" ]
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, '[skip test]')"
+
+    runs-on: ubuntu-latest
+    defaults:
+       run:
+         working-directory: ./project/backend
+    strategy:
+      matrix:
+        node-version: [18.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
```

Comment:
Requires install dependencies line node js.

 - name: Install dependencies
      run: cd <folder_name> && npm install

Reply:
Please include name: Install Dependencies or Dependency Installation just before the cache lines, in line 30.